devops/scripts/register-aad-workspace.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+: ${AAD_TENANT_ID?"You have not set you AAD_TENANT_ID in ./templates/core/.env"}
+
+LOGGED_IN_TENANT_ID=$(az account show --query tenantId -o tsv)
+CHANGED_TENANT=0
+
+if [ "${LOGGED_IN_TENANT_ID}" != "${AAD_TENANT_ID}" ]; then
+  echo "Attempting to sign you onto ${AAD_TENANT_ID} to add an App Registration."
+
+  # First we need to login to the AAD tenant (as it could be different to the subscription tenant)
```

Comment:
This comment is repeated (here and line 26). I suspect here it should say `as it is different` rather than `as it could be different`